wpeverest/user-registration
# Feature - Live new user registration notice and badge

### All Submissions:






### Changes proposed in this Pull Request:



New feature added for badge and notice for the newly registered users using the heartbeat technology of WordPress.

### How to test the changes in this Pull Request:

1. Open user listing page in admin.
2. Register a new user.
3. Badge and notice will appear in the user menu and top of the page respectively.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Live new user registration notice and badge
